week3/homework/src/todoActions.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unneeded-ternary */
+'use strict';
+
+const Todo = require('./todo');
+const util = require('./util');
+
+class TodoActions {
+  async readToDo(req, res) {
+    const todo = util.todos.find(todo => todo.id === req.params.id);
+    if (todo)
+      res.status(200).json({ message: 'Todo found', todos: todo });
+    else
+      res.status(404).json({ message: 'Todo is not found' });
+  }
+
+  async readToDos(req, res) {
+    res.status(200).json({ message: 'TODO LIST', todos: util.todos });
+  }
+
+  async createToDo(req, res) {
+    try {
+      if (req.body.description.trim()) {
+        util.todos.push(new Todo(req.body.description));
+        await util.saveTodos(util.todos);
+        res.status(200).json({ message: 'Todo is created successfully' });
+      }
+      else {
+        res.status(400).json({ message: 'Description is empty!' });
+      }
+    }
+    catch (e) {
+      res.status(400).json({ message: 'Type (description) correctly' });
+    }
+  }
+
+  async markToDo(req, res) {
+    const index = util.todos.findIndex(todo => todo.id === req.params.id);
+    const markType = req.method === 'POST' ? true : false;
+    const msg = req.method === 'POST' ? 'completed' : 'uncompleted';
```

Comment:
what about using these both together as:
```javascript
    // responsible to handle the post and delete toggling the todo item completeness.
    const isPost = req.method === 'POST' ? true : false;
    const msg = isPost ? 'completed' : 'uncompleted';
```

with a slightly descriptive comment about why `POST` it's gonna be a lot better to know what is happening.

Reply:
That is what refactoring is... Thank you :)